src/Zicht/Bundle/AdminBundle/Resources/translations/admin.en.yml

```diff
@@ -1,3 +0,0 @@
-zicht_quicklist:
-    autocomplete_type:
-        search_placeholder: Search...
```

Comment:
Dit lijkt nog wel in gebruik te zijn, of niet?
https://github.com/zicht/admin-bundle/blob/release/6.x/src/Form/AutocompleteType.php#L88

Reply:
Dit is een heel raar pad, waarschijnlijk vergeten met de PSR-4 update. De inhoud staat wel in `admin.en.yml` op een andere plek.